.gitignore

```diff
@@ -6,3 +6,19 @@ build/
 
 # editor config
 .vscode/
+
+# Xcode
+*.pbxuser
```

Comment:
as I'm not an Xcode user, I'm not sure what all of this is. what are all of these?